tesseract-server/src/logic_layer/cache.rs

```diff
@@ -225,6 +225,19 @@ impl CubeCache {
             None => None
         }
     }
+
+    // TODO note that this is being used in core tesseract, but that the cache is created using
+    // logic layer rules. This means that at the moment of this implementation, this will work in
+    // core tesseract but only if the core tesseract schema can also be a logic layer schema (and
+    // satisfying all the rules about level, property uniqueness, etc.)
```

Comment:
overall this looks great! what will happen if a user doesn't satisfy the rules about level/property uniqueness at the moment, what kind of error would they get?

Reply:
@MarcioPorto we should just review this together